repo/src/main/java/com/parashift/onlyoffice/CallBack.java

```diff
@@ -259,8 +256,21 @@ public Object execute() throws Throwable {
         }
     }
 
-    private void updateNode(NodeRef nodeRef, String url) throws Exception {
+    private void updateNode(final NodeRef nodeRef, String url) throws Exception {
         logger.debug("Retrieving URL:" + url);
+
+        final String currentUser = AuthenticationUtil.getFullyAuthenticatedUser();
+
+        AuthenticationUtil.runAs(new AuthenticationUtil.RunAsWork<Void>() {
+            public Void doWork() {
+                NodeRef sourcesNodeRef = cociService.getCheckedOut(nodeRef);
+                nodeService.setProperty(sourcesNodeRef, ContentModel.PROP_LOCK_OWNER, currentUser);
+                ownableService.setOwner(nodeRef, currentUser);
+                nodeService.setProperty(nodeRef, ContentModel.PROP_WORKING_COPY_OWNER, currentUser);
+                return null;
+            }
+        }, AuthenticationUtil.getSystemUserName());
```

Comment:
Hello,

maybe you should use **AuthenticationUtil.getAdminUserName ()** because the system user doesn't exist from versions 6.x

Regards

Reply:
Hey @jberthoux 
why do you think so?
Assembly problems do not arise in 7.1

Reply:
Hi @LinneyS ,

We have already had issues with runAsSystem during Alfresco 5 to 6 migration.
But it was only a suggestion.

Reply:
We do not support version 5